.evergreen/scripts/integration-tests.sh

```diff
@@ -178,14 +177,11 @@ fi
 if [ ! -z "$REQUIRE_API_VERSION" ]; then
   MONGO_SHELL_CONNECTION_FLAGS="${MONGO_SHELL_CONNECTION_FLAGS} --apiVersion=1"
   # Set the requireApiVersion parameter.
-  `pwd`/mongodb/bin/mongo $MONGO_SHELL_CONNECTION_FLAGS $DIR/../etc/require-api-version.js
+  mongosh $MONGO_SHELL_CONNECTION_FLAGS $DIR/../etc/require-api-version.js
```

Comment:
The test-versioned-api task [does not find mongosh](https://spruce.mongodb.com/task/mongo_c_driver_versioned_api_test_versioned_api_patch_39f56c4b01c3f98a5d28f76a8dcbd76938819d33_649cb3743066154422e07099_23_06_28_22_25_59/logs?execution=0):

```
.evergreen/scripts/integration-tests.sh: line 180: mongosh: command not found
```

I expect some form of this line from `run-orchestration.sh` may be needed to download mongosh in `integration-tests.sh`:
```
DRIVERS_TOOLS=./ download_and_extract "$MONGODB_DOWNLOAD_URL" "$EXTRACT" "$MONGOSH_DOWNLOAD_URL" "$EXTRACT_MONGOSH"
```

And (possibly?) specify the path to `mongosh`: 

```suggestion
  `pwd`/mongodb/bin/mongosh $MONGO_SHELL_CONNECTION_FLAGS $DIR/../etc/require-api-version.js
```